maintenance_equipment_scrap/wizard/scrap_equipment.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_perform_equipment_scrap" model="ir.ui.view">
+        <field name="model">wizard.perform.equipment.scrap</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="equipment_id" readonly="1" string="Scrapping equipment"/>
+                            <field name="scrap_date" default_focus="1" />
+                        </group>
+                    </group>
+                </sheet>
+                <footer>
+                    <button name="do_scrap" string="Scrap" type="object" class="oe_highlight" />
+                    <button special="cancel" string="Cancel" class="btn-default"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="wizard_perform_equipment_scrap_action">
```

Comment:
optional change:
id before model

Reply:
This suggestion actually doesn't change anything but code style, so I preferred preserve the original code in the file history